src/components/SquigglyTitle.tsx

```diff
@@ -0,0 +1,149 @@
+import styled from 'styled-components';
+import { FONT_DISPLAY, MOBILE_MAX_WIDTH, TABLET_MAX_WIDTH } from '~/components';
+
+interface Props {
+  text: string; // use '\n' to break lines in a template literal
+  sizes: titleSizes;
+}
+
+type remUnit = `${number}rem`;
+type vwUnit = `${number}vw`;
+
+type titleSizes = {
+  lg: remUnit;
+  md: remUnit;
+  sm: remUnit;
+  lgvw: vwUnit;
+  mdvw: vwUnit;
+  smvw?: vwUnit;
+};
+
+export const SquigglyTitle = ({ text, sizes }: Props) => {
+  sizes.smvw ??= sizes.mdvw; // fallback to mdvw if smvw is not provided
+
+  return (
+    <StyledWrapper>
+      <StyledTitle $sizes={sizes}>{text}</StyledTitle>
+
+      <StyledSVG xmlns='http://www.w3.org/2000/svg' version='1.1' width='102%' height='100%'>
+        <defs>
+          {/* Squiggly filter desktop and tablet */}
+          <filter
+            id='squiggly-filter'
+            x='-20%'
+            y='-20%'
+            width='140%'
+            height='140%'
+            filterUnits='objectBoundingBox'
+            primitiveUnits='userSpaceOnUse'
+            colorInterpolationFilters='linearRGB'
+          >
+            <feTurbulence
+              type='fractalNoise'
+              baseFrequency='0.02 0.0125'
+              numOctaves='3'
+              seed='2'
+              stitchTiles='stitch'
+              x='0%'
+              y='0%'
+              width='200%'
+              height='100%'
+              scale='1'
+              result='turbulence'
+            />
+            <feDisplacementMap
+              in='SourceGraphic'
+              in2='turbulence'
+              scale='5'
+              xChannelSelector='R'
+              yChannelSelector='B'
+              x='0%'
+              y='0%'
+              width='100%'
+              height='100%'
+              result='displacementMap'
+            >
+              <animate attributeName='scale' values='5;15;5;' dur='2' repeatCount='indefinite' begin='0' />
+            </feDisplacementMap>
+          </filter>
+          {/* Squiggly filter mobile */}
```

Comment:
are the comments necessary given that the ID already indicates whether it's for mobile or desktop?

Reply:
I think it helps not to be scared with a lot of uncommon tags 😅 and quickly answer why the filter is "duplicated".
In a next PR could be simplified to just one filter and modify the attributes with JS for mobile.